docs/developer-docs/release-procedure.md

```diff
@@ -0,0 +1,77 @@
+# Release Procedure
+
+This document is a Core team guide, provided publicly to ensure transparancy in the release process.
+
+## Versioning
+
+Jellyfin uses a 3-digit semantic versioning scheme. All releases will have versions in the `X.Y.Z` format, starting from `10.0.0`. Note however that the `10.X.Y` release chain represents the "cleanup" of the codebase, so it should be accepted that `10.X.Y` breaks all compatibility, at some point, with previous Emby-compatible interfaces.
```

Comment:
Do you mean to use “10.X.Y”, or should that be “10.Y.Z”?

Reply:
Corrected for consistency; not intentional.